modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalArticleLocalServiceTest.java

```diff
@@ -1096,6 +1097,44 @@ public void testDuplicatedAutoGeneratedArticleId() throws Exception {
 			article.getArticleId(), true);
 	}
 
+	@Test
+	public void testFetchArticleByPrimaryKey() throws Exception {
+		JournalArticle article = JournalTestUtil.addArticle(
+			_group.getGroupId(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID);
+
+		PersistedModel persistedModel =
+			_journalArticleLocalService.fetchPersistedModel(
+				article.getResourcePrimKey());
+
+		Assert.assertNotNull(persistedModel);
+	}
+
+	@Test
+	public void testFetchLastApprovedVersionByPrimaryKey() throws Exception {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				_group.getGroupId(), TestPropsValues.getUserId());
+
+		JournalArticle approvedJournalArticle = JournalTestUtil.addArticle(
```

Comment:
@manuelRives, we can name just "article"

Reply:
Fixed, Good catch!

Thanks for reviewing it!